opentreemap/treemap/js/src/CanopyFilterControl.js

```diff
@@ -0,0 +1,56 @@
+"use strict";
+
+var $ = require('jquery'),
+    _ = require('lodash'),
+    L = require('leaflet'),
+    Bacon = require('baconjs');
+
+require('ionRangeSlider');
+
+var canopyFilterTmpl = _.template($('#canopy-filter-tmpl').html());
+
+var CanopyFilterControl = L.Control.extend({
+    options: {
+        position: 'bottomleft'
+    },
+
+    initialize: function() {
+        this._changeBus = new Bacon.Bus();
+        this.tilerArgsProp = this._changeBus.toProperty({
+            category: 'Municipality',
```

Comment:
If we opt to hard code a value, we will need to remove this configuration field from the model, to avoid confusion.

```python
    """
    The boundary category to be used for showing a choropleth canopy
    layer. max_length=255 matches Boundary.category
    """
    canopy_boundary_category = models.CharField(max_length=255, default='',
                                                blank=True)
```

https://github.com/OpenTreeMap/otm-core/blob/9f49ddf638a0ce8ecac0ace2a02f5a0b641edd71/opentreemap/treemap/instance.py#L225-L230

I am ok with removing it.